kartothek/serialization/_generic.py

```diff
@@ -378,7 +378,7 @@ def _ensure_type_stability(
         error messages.
     """
 
-    value_dtype = pd.Series(value).dtype
+    value_dtype = pd.Series(value if is_list_like(value) else [value]).dtype
```

Comment:
Is this user facing, i.e. is it relevant enough to mention in a changelog?

Reply:
Do we mention fixed warnings in the changelog? I'm OK with this, just wanted to clarify.

Reply:
If it's just about the warning, I don't care at all. I thought it was a fix since you added another test

Reply:
Valid assumption/question. Added the other test to make sure it is working (like a safety net). The test status didn't change due to the actual change here.